### DISCUSSION TITLE
The skin editor unexpectedly modifies skins

### DISCUSSION THREAD

**Discussion by soopax** [ppy/osu#23735] [Q&A] [answered]
# Steps to reproduce

1. Change the current skin to the default.
2. Play beatmap 1 with autoplay and pause the playback.
3. Open the skin editor. This should create a new skin based off the default skin.
4. In the list of components, click on "Big Black Box".
5. Save the skin, then close the skin editor.
6. Observe what the skin looks like currently.
7. Go back to the song select, play beatmap 2 with autoplay and pause the playback.
8. Open the skin editor.
9. Close the skin editor.
10. Go back to the song select and play beatmap 1 again.
11. The skin looks different.

## Expected result
The skin should look the same in steps 6 and 11, because the skin wasn't edited after step 5.

## Actual result
The Big Black Box is gone and the skin looks messed up.

# Files
Beatmap 1: https://osu.ppy.sh/beatmapsets/949297#osu/1982307
Beatmap 2: https://osu.ppy.sh/beatmapsets/883088#osu/1846145
Video: Due to a Github limit, please rename the file extension `.txt` to `.mkv`. [2023-06-03_07-52-26.txt](https://github.com/ppy/osu/files/11641763/2023-06-03_07-52-26.txt)

# Edit:
Added a video and changed the formatting.

**Comment by peppy** [comment]
Please include a video.

**Reply by soopax** [reply]
Sorry for any inconvenience. Here is a video. Due to a Github limit, please rename the file extension `.txt` to `.mkv`.
[2023-06-03_07-52-26.txt](https://github.com/ppy/osu/files/11641763/2023-06-03_07-52-26.txt)

**Comment by bdach** [comment, marked as answer]
duplicate of https://github.com/ppy/osu/issues/22561

**Reply by soopax** [reply]
Oops, I tried searching but I couldn't find this.
